tests/test_termui.py

```diff
@@ -446,3 +446,13 @@ def cli(password):
 
     if prompt == "Confirm Password":
         assert "Confirm Password: " in result.output
+
+
+def test_show_default_false_hides_prompt(runner):
```

Comment:
This test name ican be clearer in order to explain what does this test check.

Mybe something like "test_false_show_default_cause_no_default_display_in_prompy".

Reply:
Done